app/controllers/hubstats/deploys_controller.rb

```diff
@@ -27,6 +22,41 @@ def index
     end
 
     def show
+      @deploy = Hubstats::Deploy.find(params[:id])
+      @repo = @deploy.repo
+      @pull_requests = @deploy.pull_requests
+      @pull_request_count = @pull_requests.length
+      @net_additions = find_net_additions(@deploy.id)
+      @comment_count = find_comment_count(@deploy.id)
+      @stats = {
+        pull_count: @pull_request_count,
+        net_additions: @net_additions,
+        comment_count: @comment_count
+      }
+    end
+
+    def find_net_additions(deploy_id)
```

Comment:
Use local not instance scoped variables in this method and `find_comment_count`.  You are just using them for their return value.